pkg/gofr/container/datasources.go

```diff
@@ -174,3 +180,7 @@ type provider interface {
 	// Connect establishes a connection to Cassandra and registers metrics using the provided configuration when the client was Created.
 	Connect()
 }
+
+type HealthChecker interface {
+	HealthCheck() (any, error)
```

Comment:
Are you referring to a Health struct?
If yes, then we are returning interface because externalDB's have their own module because we dont want to keep a dependency if user doesn't want to use it.
It would only increase the binary size of the application.

Reply:
OK it's clearer. Then maybe a comment about it on the interface would be great.

It's a valuable information